app/PrettyAnsi.hs

```diff
@@ -0,0 +1,37 @@
+module PrettyAnsi where
+
+ansiBlack :: [Char]
+ansiBlack = "\x1b[30m"
+ansiRed :: [Char]
+ansiRed = "\x1b[31m"
+ansiGreen :: [Char]
+ansiGreen = "\x1b[32m"
+ansiYellow :: [Char]
+ansiYellow = "\x1b[33m"
+ansiBlue :: [Char]
+ansiBlue = "\x1b[34m"
+ansiMagenta :: [Char]
+ansiMagenta = "\x1b[35m"
+ansiCyan :: [Char]
+ansiCyan = "\x1b[36m"
+ansiWhite :: [Char]
+ansiWhite = "\x1b[37m"
+ansiReset :: [Char]
+ansiReset = "\x1b[0m"
+ansiBold :: [Char]
+ansiBold = "\x1b[0m"
+
+prependColor :: [[Char]] -> Int -> (Int, [Char]) -> [Char]
+prependColor colors colorsCount (number, word)= do
```

Comment:
Actually, `do` is more of a "deal with monads" syntax, similar to `for` comprehensions in Scala, as I understand it. The Haskell way to put this would be something like
```haskell
                              // number - is this the same as "word index"?
prependColor colors colorsCount (number, word) =
    color ++ word
    where
        colorIndex = mod number colorsCount
        color =  colors !! colorIndex
```

also, check out this: 
```haskell
Prelude> borges = ["Banshees", "Celestial Cock", "Chinese Dragon", "Western Dragon", "Hydra", "Lilith", "Golem", "Unicorn"]

Prelude> rasta = ["Red", "Golden", "Green","Black"]
Prelude> take 10 $ cycle rasta
["Red","Golden","Green","Black","Red","Golden","Green","Black","Red","Golden"]

Prelude> zip rasta borges
[("Red","Banshees"),("Golden","Celestial Cock"),("Green","Chinese Dragon"),("Black","Western Dragon")]

Prelude> [color ++ " " ++ beast | (color, beast) <- zip rasta borges]
["Red Banshees","Golden Celestial Cock","Green Chinese Dragon","Black Western Dragon"]

Prelude> [color ++ " " ++ beast | (color, beast) <- zip (cycle rasta) borges]
["Red Banshees","Golden Celestial Cock","Green Chinese Dragon","Black Western Dragon","Red Hydra","Golden Lilith","Green Golem","Black Unicorn"]
```